public/resources/default-files/html.txt

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Made with Thimble</title>
-</head>
-<body>
-
-  <h1>Welcome to Thimble</h1>
-
-  <p>Make something <strong>amazing</strong> with the web!</p>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Made with Thimble</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
```

Comment:
@flukeout It doesn't appear these changes have landed on staging yet, but this specific line wasn't present on the original in comparison to the diff, but when I create a new project I see this link added in. Are we going to have two links to style.css now?